pkg/scheduler/api/v1/types.go

```diff
@@ -137,17 +137,17 @@ type RequestedToCapacityRatioArguments struct {
 // UtilizationShapePoint represents single point of priority function shape.
 type UtilizationShapePoint struct {
 	// Utilization (x axis). Valid values are 0 to 100. Fully utilized node maps to 100.
-	Utilization int `json:"utilization"`
+	Utilization int8 `json:"utilization"`
```

Comment:
I don't think we should change it to int8 which is not common in the k/k

```
$ ag int8 pkg
pkg/controller/daemon/daemon_controller_test.go
580:			for pass := uint8(0); expectedLimit <= podControl.FakePodControl.CreateLimit; pass++ {

pkg/controller/job/job_controller_test.go
360:		for pass := uint8(0); expectedLimit <= tc.podLimit; pass++ {

pkg/controller/replicaset/replica_set_test.go
269:	for pass := uint8(0); expectedLimit <= fakePodControl.CreateLimit; pass++ {

pkg/kubelet/apis/podresources/v1alpha1/api.pb.go
398:				dAtA[i] = uint8(uint64(l)&0x7f | 0x80)
402:			dAtA[i] = uint8(l)
412:		dAtA[offset] = uint8(v&0x7f | 0x80)
416:	dAtA[offset] = uint8(v)

pkg/kubelet/apis/pluginregistration/v1/api.pb.go
309:				dAtA[i] = uint8(uint64(l)&0x7f | 0x80)
313:			dAtA[i] = uint8(l)
393:		dAtA[offset] = uint8(v&0x7f | 0x80)
397:	dAtA[offset] = uint8(v)

pkg/kubelet/apis/pluginregistration/v1alpha1/api.pb.go
560:		dAtA[offset] = uint8(v&0x7f | 0x80)
564:	dAtA[offset] = uint8(v)

pkg/kubelet/apis/pluginregistration/v1beta1/api.pb.go
560:		dAtA[offset] = uint8(v&0x7f | 0x80)
564:	dAtA[offset] = uint8(v)

pkg/kubelet/apis/deviceplugin/v1alpha/api.pb.go
1169:		dAtA[offset] = uint8(v&0x7f | 0x80)
1173:	dAtA[offset] = uint8(v)

pkg/kubelet/apis/deviceplugin/v1beta1/api.pb.go
1857:		dAtA[offset] = uint8(v&0x7f | 0x80)
1861:	dAtA[offset] = uint8(v)

pkg/kubelet/pluginmanager/pluginwatcher/example_plugin_apis/v1beta2/api.pb.go
324:		dAtA[offset] = uint8(v&0x7f | 0x80)
328:	dAtA[offset] = uint8(v)

pkg/kubelet/pluginmanager/pluginwatcher/example_plugin_apis/v1beta1/api.pb.go
323:		dAtA[offset] = uint8(v&0x7f | 0x80)
327:	dAtA[offset] = uint8(v)

pkg/registry/core/service/allocator/utils.go
31:var bitCounts = []int8{

pkg/kubectl/cmd/get/sorter.go
174:	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
176:	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
211:		case uint8:
212:			if jtype, ok := j.Interface().(uint8); ok {
227:		case int8:
228:			if jtype, ok := j.Interface().(int8); ok {
```

Reply:
Do you think it's better to use int32 like other fields even if it's bounded by (0-100)?

Reply:
Done